mart/models/modular.py

```diff
@@ -232,11 +232,15 @@ def log_metrics(self, metrics, prefix="", prog_bar=False):
         def enumerate_metric(metric, name):
             # Metrics can have arbitrary depth.
             if isinstance(metric, torch.Tensor):
-                metrics_dict[name] = metric
+                # Ignore non-scalar results generated by Metrics, such as list of classes from MAP.
```

Comment:
It would be nice to flatten so we get per class AP numbers. This doesn't work right now though and just returns the average 🙃

Reply:
Good point. I would make per-class AP in a separate PR.